website/docs/r/ec2_client_vpn_endpoint.html.markdown

```diff
@@ -36,15 +36,16 @@ resource "aws_ec2_client_vpn_endpoint" "example" {
 
 The following arguments are supported:
 
-* `authentication_options` - (Required) Information about the authentication method to be used to authenticate clients.
```

Comment:
Was there a particular reason why this was moved? It was previously alphabetical.

Reply:
I left the original author's edit